packages/flutter/lib/src/widgets/widget_inspector.dart

```diff
@@ -935,6 +935,42 @@ mixin WidgetInspectorService {
     return Future<void>.value();
   }
 
+  static const String consoleObjectGroup = 'console-group';
+
+  void _reportError(FlutterErrorDetails details) {
```

Comment:
this CL turns on postEvent everywhere. Next step is to 
add methods for clients to start listening for the events and methods for clients to clear the console.
Currently there is one hard coded consoleObjectGroup but that would be a bit problematic if multiple clients were listening and one wants to clear.